src/_scss/pages/agencyV2/statusOfFunds/_introSection.scss

```diff
@@ -0,0 +1,38 @@
+.status-of-funds__intro-wrapper {
+    @include span-columns(16);
+    text-align: left;
+    margin: auto;
+    padding-bottom: 4rem;
+    @media (min-width: $medium-screen) {
+        padding-left: 4.8rem;
+    }
+}
+
+.status-of-funds__intro-section-title {
+    font-size: 2rem;
+    color: #555555;
```

Comment:
We could use one of our color variables here, `$color-gray` or `$color-base`. (This also might be the default color and this rule is unnecessary)

Reply:
Yep, the color rule isn't necessary here at all. Removed.